test/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+		<title>d2l-hierarchical-view tests</title>
+		<script src="../../web-component-tester/browser.js"></script>
+	</head>
+	<body>
+		<script>
+		WCT.loadSuites([
+			'visibility.html',
+			'visibility.html?dom-shadow',
+			'focus-behavior.html',
+			'focus-behavior.html?dom=shadow'
```

Comment:
Cool thanks.  Yeah.  I've been running them explicitly in browser so I missed updating this.